apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/IterableContainsInOrderOnlyElementsOfAssertionSpec.kt

```diff
@@ -4,33 +4,33 @@ import ch.tutteli.atrium.creating.Expect
 import org.spekframework.spek2.Spek
 import kotlin.reflect.KFunction3
 
-class IterableContainsInOrderElementsOfAssertionSpec : Spek({
+class IterableContainsInOrderOnlyElementsOfAssertionSpec : Spek({
     include(BuilderSpec)
 }) {
-    object BuilderSpec : ch.tutteli.atrium.specs.integration.IterableContainsInOrderOnlyEntriesAssertionsSpec  (
+    object BuilderSpec : ch.tutteli.atrium.specs.integration.IterableContainsInOrderOnlyValuesAssertionsSpec  (
         getContainsPair(),
         getContainsNullablePair(),
-        "◆ ", "✔ ", "✘ ", "❗❗ ", "⚬ ", "» ", "▶ ", "◾ ",
+        "◆ ", "✔ ", "✘ ", "❗❗ ", "⚬ ", "▶ ", "◾ ",
         "[Atrium][Builder] "
     )
 
     companion object : IterableContainsSpecBase() {
         fun getContainsPair() = "$contains.$inOrder.$inOrderElementsOf" to Companion::containsInOrderOnlyValues
 
         private fun containsInOrderOnlyValues(
-            expect: Expect<Iterable<Double>>,
-            a: Expect<Double>.() -> Unit,
-            aX: Array<out Expect<Double>.() -> Unit>
-        ): Expect<Iterable<Double>> = expect.contains.inOrder.only.elementsOf(listOf(a, *aX))
+            plant: Expect<Iterable<Double>>,
```

Comment:
Keep `expect` instead of plant

Reply:
Done! What is the difference between the two? I though plant was for "in order" and "expected" was for "any order" but perhaps I was wrong.

Reply:
it's just naming really, plant was used for Assert/AssertionPlant which is deprecated. Thus I don't want to use it for Expect.